tutorials/iot/ubuntu-kiosk/electron-kiosk.md

```diff
@@ -180,7 +179,7 @@ and on your desktop, install, connect the required interface(s) and run:
 
 ```bash
 sudo snap install --dangerous ./electron-hello-world-kiosk_0.1_amd64.snap
-sudo snap connect electron-hello-world-kiosk:browser-support
+sudo snap connect electron-hello-world-kiosk:wayland
```

Comment:
This is missing `mir-kiosk` at the end, no?
```
$ snap connect electron-hello-world-kiosk:wayland
error: snap "snapd" has no "wayland" interface slots
```

Reply:
You should not get this error here (on desktop). Are you sure?

Reply:
Right, was testing this on Core, but… the above YAML doesn't even have a `wayland` plug?